ocaml/xapi-cli-server/cli_frontend.ml

```diff
@@ -1047,6 +1047,15 @@ let rec cmdtable_data : (string * cmd_spec) list =
       ; flags= [Neverforward]
       }
     )
+  ; ( "host-show-updates"
```

Comment:
I see `xe host-available-updates` in the design: https://code.citrite.net/projects/XSC/repos/xs-documents/browse/designs/on-prem-cdn-update/implementation.md?useDefaultHandler=true#379

Reply:
I think `host-show-updates` complies with the convention of <object>-<verb>-<noun>.

Reply:
Agree, then how about `host-show-available-updates`? I think it will be easier to understand.

Reply:
This is difficult to discover using auto complete. So I would suggest `host-updates-show-avaiable` The reason is that a user wants to know something about updates, so will start from `host-updates-…` to see available commands. The noun in this case is `host-updates`.

Reply:
How about the `host-apply-updates`? Then rename it to `host-updates-apply`?

Reply:
I would find that better discoverable; however, if the command already exists the change would break existing code. In that case having a second name could be a choice to improve discoverability.